doc/source/tutorials/go_models.rst

```diff
@@ -0,0 +1,133 @@
+    with strength 9.414 kJ/mol (changed through the ``-go-eps`` flag).
+ 3) Eliminate any contacts which are shorter than 0.3 nm and longer than 1.1 nm, or are closer than 3 residues in the
+    molecular graph. These options are flexible through the ``-go-low`` and ``-go-up`` flags.
+ 4) If the contact map finds any atoms within contact range defined, but are *also* within 3 residues of each other,
+    then the contacts are removed. This is defined through the ``-go-res-dist`` flag.
+
+As a result, along with the standard output of martinize2 (*i.e.* itp files for your molecules, a generic .top file,
+a coarse grained structure file), you will get two extra files: ``go_atomtypes.itp`` and ``go_nbparams.itp``. The atomtypes
+file defines the new virtual sites as atoms for your system, and the nbparams file defines specific non-bonded
+interactions between them.
+
+For example, ``go_atomtypes.itp`` looks like any other ``[ atomtypes ]`` directive::
+
+ [ atomtypes ]
+ molecule_0_1 0.0 0 A 0.00000000 0.00000000
+ molecule_0_2 0.0 0 A 0.00000000 0.00000000
+ molecule_0_3 0.0 0 A 0.00000000 0.00000000
+ molecule_0_4 0.0 0 A 0.00000000 0.00000000
+ molecule_0_5 0.0 0 A 0.00000000 0.00000000
+ ...
+
+Similarly, ``go_nbparams.itp`` looks like any ``[ nonbond_params ]`` directive (obviously, the exact parameters here
+depend on your protein)::
+
+ [ nonbond_params ]
+ molecule_0_17 molecule_0_13 1 0.59354169 9.41400000 ;go bond 0.666228018941817
+ molecule_0_18 molecule_0_14 1 0.53798937 9.41400000 ;go bond 0.6038726468003999
+ molecule_0_19 molecule_0_15 1 0.51270658 9.41400000 ;go bond 0.5754936778307316
+ molecule_0_22 molecule_0_15 1 0.73815666 9.41400000 ;go bond 0.8285528398039018
+ molecule_0_22 molecule_0_18 1 0.54218134 9.41400000 ;go bond 0.6085779754055839
+ molecule_0_23 molecule_0_19 1 0.53307395 9.41400000 ;go bond 0.5983552758317587
+ ...
+
+To activate your Go model for use in Gromacs, the `martini_v3.0.0.itp` master itp needs the additional files included.
+The additional atomtypes defined in the ``go_atomtypes.itp`` file should be included at the end of the `[ atomtypes ]`
+directive as::
+
+
+ [ atomtypes ]
+ ...
+ TX1er 36.0 0.000 A 0.0 0.0
+ W  72.0 0.000 A 0.0 0.0
+ SW 54.0 0.000 A 0.0 0.0
+ TW 36.0 0.000 A 0.0 0.0
+ U  24.0 0.000 A 0.0 0.0
+
+ #ifdef GO_VIRT
+ #include "go_atomtypes.itp"
+ #endif
+
+ [ nonbond_params ]
+ P6    P6  1 4.700000e-01    4.990000e+00
+ P6    P5  1 4.700000e-01    4.730000e+00
+ P6    P4  1 4.700000e-01    4.480000e+00
+ ...
+
+Similarly, the nonbonded parameters should be included at the end of the `[ nonbond_params ]`
+directive::
+
+ ...
+ TX2er  SQ1n  1 3.660000e-01    3.528000e+00
+ TX2er  TQ1n  1 3.520000e-01    5.158000e+00
+ TX1er   Q1n  1 3.950000e-01    1.981000e+00
+ TX1er  SQ1n  1 3.780000e-01    3.098000e+00
+ TX1er  TQ1n  1 3.660000e-01    4.422000e+00
+
+ #ifdef GO_VIRT
+ #include "go_nbparams.itp"
```

Comment:
@csbrasnett should we perhaps update the master itp supplied via the new website?

Reply:
I agree, have opened an issue [here](https://github.com/Martini-Force-Field-Initiative/Martini-Force-Field-Initiative.github.io/issues/41)